conflux/caip2.md

```diff
@@ -0,0 +1,90 @@
+---
+namespace-identifier: conflux-caip2
+title: Conflux Namespace - Chains
+author: iosh (@iosh)
+discussions-to: <URL of PR, mailing list, etc>
+status: Draft
+type: Standard
+created: 2024-08-09
+requires: CAIP-2
+---
+
+# CAIP-2
+
+_For context, see the [CAIP-2][] specification._
+
+## Rationale
+
+Conflux network(core space) consists of 2 networks: main network and testing network. The private network can also be created , Each network has a unique network ID that can be used to identify it.
```

Comment:
Do private networks have to register their network ID in a registry to prevent collisions, or is there a "vendor range" (as it would be called at IETF) for arbitrary chainIds to be claimed informally, and published out-of-band between users?  If there is a formal registry, please link to it here.

Reply:
We have only defined two network ids, Mainnet 1029 and Testnet 1.  Private networks do not have a registry to prevent collisions and currently private netwoks are mostly used for local test.